### ISSUE TITLE
Prefer built-in BT radio for pairing, if present

### ISSUE THREAD

**Issue by adriangranados** [WLAN-Pi/wlanpi-bluetooth#1]
When using a Raspberry Pi 4, for example, the built-in BT radio may show up as a second BT interface if a USB adapter that supports BT is also connected. In other words, the built-in BT radio gets hci1 and the external radio gets hci0. Since we use hci0 for pairing, it may not work. We need to figure out a way to prefer the built-in BT radio as we know it works.

**Comment by meanmrgreen** [comment]
Yeah im having the exact problem right now.. can pair with the pi4 without wifi dongle attached but not when i attach it 😁

**Comment by jiribrejcha** [comment]
It will be fixed in 3.1.3 release. Beta version of wlanpi-common package, which includes the fix, is now available for testing:

1. Download the package:
`wget --content-disposition "https://packagecloud.io/wlanpi/dev/packages/debian/bullseye/wlanpi-common_1.0.88_arm64.deb/download.deb?distro_version_id=207"`
2. Install the package:
sudo apt install ./wlanpi-common_1.0.88_arm64.deb

**Comment by joshschmelzle** [comment]
Not sure how this was fixed, but closing this issue.
